### ISSUE TITLE
How do I set a custom node path?

### ISSUE THREAD

**Issue by petercunha** [tanepiper/SublimeText-Nodejs#60]
I'm getting an error when ever I try to use a utility from the Tools --> NodeJS menu:
```
Node binary could not be found in PATH

Consider using the node_command setting for the Node plugin

PATH is: /usr/bin:/bin:/usr/sbin:/sbin
```

I ran `which node` in terminal and got `/usr/local/bin/node`. How do I set that as the default node path?

**Comment by techaware** [comment]
I am also facing the same issue.

**Comment by varp** [comment]
@petercunha @techaware You can set the default node path either by Preferences --> Package Settings --> NodeJs --> User Settings or add `/usr/local/bin` to your `PATH`

**Comment by eselkin** [comment]
As Packages/User/Nodejs.sublime-settings
{
	"node_command": "/usr/local/bin/node",
	"node_path": "/usr/local/bin/node",
	"node" : "/usr/local/bin/node"
}
Using this setting or any of those individually do not work.
Nor does setting path in Preferences.sublime-settings

**Comment by JethroSovr** [comment]
I used brew to install the NodeJs
type "where node" in terminal,
it showed  "/usr/local/bin/node"
so I also got this problem ,
I try to setting like eselkin by edit "Nodejs.sublime-settings"
but it doesnt work.
So my fix like this:
edit the "Nodejs.sublime-build"
"osx":
    {
        "cmd": ["killall node;/usr/local/bin/node $file"]
    }
use "/usr/local/bin/node"  to replace "node" 
now it worked, but I think it must have the  another way to fix it.

**Comment by varp** [comment]
I'm digging the issue. Will give feedback asap

**Comment by natanavra** [comment]
Any news on this one?

**Comment by varp** [comment]
@natanavra till May, 2 I will give a feedback

**Comment by varp** [comment]
@petercunha @techaware @eselkin @JethroSovr @natanavra please install plugin form the branch issue-60 to check if the issue is resolved

**Comment by eselkin** [comment]
Seems like it runs, but there is no output on the run. I do see its process running after using the run menu item.

**Comment by varp** [comment]
@eselkin I've heard you. I'll be working on this and will give you feedback. As always in couple of days...

**Comment by varp** [comment]
Fixed in a new release v1.5.5. Check out the master branch or install the new version from package manager.

**Comment by MadCowWeb** [comment]
I used HomeBrew and found mine here:
/Users/<your-name-here>/.nvm/versions/node/v10.16.3/bin/node
